Test/CoreSDK.Test/Shared/DataContracts/PerformanceCounterTelemetryTest.cs

```diff
@@ -8,6 +8,8 @@
     [TestClass]
     public class PerformanceCounterTelemetryTest
     {
+#pragma warning disable 618
```

Comment:
I marked class as `Obsolete` and test code wouldn't compile if you keep reference on those classes